### PR TITLE
Workspaces launch changes

### PR DESCRIPTION
**Pull request by jsdbroughton**
1. **General Refinement Across Workspace Documentation:**
    - Removed repetitive and obvious instructions (e.g., "how to create a project").
    - Focused on advanced concepts and Workspace-specific features.

2. **Integration of Speckle Automate:**
    - Introduced Private Automate Functions as a key advanced feature in Workspaces.
    - Clarified role-based permissions for automation (Admins, Members, and Guests).

3. **Updated Member Roles and Permissions:**
    - Added detailed role descriptions and clarified permissions for Admins, Members, and Guests.
    - Highlighted Contributor and Reviewer roles within projects and their access to automation.

4. **Streamlined "Workspace Projects" Section:**
    - Removed step-by-step instructions for project creation and focused on advanced features, including project-specific automations and migration.
    - Addressed data region migration (noting it as a coming feature).

5. **"Advanced Features" Page:**
    - A dedicated page was added highlighting current advanced features, specifically Private Automate Functions.
    - Future-proofed the structure for additional advanced features as they are introduced.

6. **Consistency and Tone Adjustments:**
    - Simplified marketing-style language to align with technical documentation.
    - Ensured content is concise and focused on advanced operations.

### CODE REVIEW

File: workspaces/billing.md [hunk 1]
```diff
@@ -1,10 +1,40 @@
...
+# Billing  
 
-## Workspaces are free while in beta
+Workspaces require a paid subscription to access advanced features, team collaboration tools, and larger-scale operations. This page explains the key billing concepts and operations for managing your Workspace subscription.  
 
-While Speckle is offering Workspaces for free during the beta phase, it will be a paid feature in the future. This section will detail the billing and subscription information for Workspaces.
+## Trial Period  
 
-While in beta, the billing system will give a cost estimate based on the number of members in the workspace. This will be updated to a monthly subscription model in the future.
+All new Workspaces begin with a **31-day trial** of the **Starter Plan**. During the trial:
```
Comment (benjaminvo) [inline workspaces/billing.md:7]: It's 30 days

File: workspaces/billing.md [hunk 2]
```diff
@@ -1,10 +1,40 @@
...
+- Subscription costs are calculated based on the **current member count** at the end of the billing cycle.  
+- When members are removed, their seat remains allocated and billed until the next billing cycle begins.  
+- Plans and pricing details are displayed directly in the in-app billing section.  
+- Billing supports both monthly and annual payment options.  
+
+## Managing Billing  
+
+You can manage your Workspace subscription directly from the in-app billing interface. Available operations include:  
+- **Upgrading Plans**: Easily switch to higher-tier plans as your needs evolve.  
+- **Downgrading Plans**: Not yet supported in-app; contact Speckle Support to downgrade.
```
Comment (benjaminvo) [inline workspaces/billing.md:26]: Should we mention billing@speckle.systems directly here?

File: workspaces/billing.md [hunk 3]
```diff
@@ -1,10 +1,40 @@
...
+- When members are removed, their seat remains allocated and billed until the next billing cycle begins.  
+- Plans and pricing details are displayed directly in the in-app billing section.  
+- Billing supports both monthly and annual payment options.  
+
+## Managing Billing  
+
+You can manage your Workspace subscription directly from the in-app billing interface. Available operations include:  
+- **Upgrading Plans**: Easily switch to higher-tier plans as your needs evolve.  
+- **Downgrading Plans**: Not yet supported in-app; contact Speckle Support to downgrade.  
+- **Adding or Removing Members**: Costs adjust dynamically based on the **current member count** at the end of the billing period.
```
Comment (benjaminvo) [inline workspaces/billing.md:27]: > at the end of the billing period.

Same as above. More accurate to write that it's at the start of a new billing period.

File: workspaces/billing.md [hunk 4]
```diff
@@ -1,10 +1,40 @@
...
+- At the end of the trial, Workspaces enter **read-only mode** if a subscription is not activated.  
 
-Once the beta period ends, workspaces are still free up to 500 model versions.
-To store more versions across your projects you will need to upgrade to a paid plan.
+## Subscriptions  
+
+To continue using a Workspace beyond the trial period, you will need to subscribe to a paid plan. All billing and subscription options are managed directly in the Workspace **Settings > Billing** section.  
+
+Key points:  
+- Subscription costs are calculated based on the **current member count** at the end of the billing cycle.
```
Comment (benjaminvo) [inline workspaces/billing.md:17]: More accurate to write that it's at the start of a new billing period. But we can probably also just write this in the first bullet point:
- Subscription costs adjust dynamically as you add and remove members.

File: workspaces/billing.md (hunk 3 above)
Comment (jsdbroughton) [reply workspaces/billing.md:27]: @benjaminvo Let's be crystal clear. 

Adding/Removing a member only raises/lowers cost at the start of the next billing cycle
or
Switching to a new plan goes live at the start of a new billing cycle

? both are in your highlighted section

Comment (benjaminvo) [reply workspaces/billing.md:27]: Sorry for the confusion. I'll try to be crystal clear:

- Adding a member immediately increases the subscription and cost by 1 seat (if there are no unoccupied seats).
- Removing a member keeps the the current seat count until the current billing cycle ends. Seats are recalculated based on current member count when a new billing cycle starts.
- Switching to a new plan goes live immediately.

**Pull request merged**